### PR TITLE
update README to clarify dependences and setup

### PR DESCRIPTION
**Pull request by peetucket**

### CODE REVIEW

**Comment by jcoyne** [comment]
@peetucket needs a rebase.

**Comment by peetucket** [comment]
rebased

File: README.md [hunk 1]
```diff
@@ -77,9 +80,9 @@ Also see:
 
 ```bash
 # Preferred
-$ rake
-
-# Run all the tests without automatically starting jetty.
+$ rake # Run all tests. 
+  
+# Run all the tests including starting/stopping jetty and refreshing all fixtures.
```
Comment (jcoyne) [inline README.md]: Is this correct? I thought local_ci didn't do this.

Comment (peetucket) [inline README.md]: looks like it does in the definition of the task as it runs several other rake tasks including the ci one: https://github.com/sul-dlss/hydrus/blob/master/lib/tasks/ci.rake#L33

Comment (jcoyne) [inline README.md]: Okay, I guess that makes this correct.  But I'd like to get rid of that task (in a separate PR). I don't know why you would want to use it that way.

Comment (peetucket) [inline README.md]: agreed, doesn't seem terribly useful

**Pull request merged**